bazadanych2aa.py

```diff
@@ -0,0 +1,116 @@
+#SQLAlchemy jest zbiorem narzędzi do pracy z bazami danych i Pythonem.
```

Comment:
Czemu to się nazywa `bazadanych2aa.py` ?? Może po prostu `bazadanych.py` albo jeszcze lepiej `catalog.py`